lib/broccoli/ember-app.js

```diff
@@ -1084,10 +1084,30 @@ class EmberApp {
     }
     let addons = this.addonTreesFor('styles');
 
-    return mergeTrees(addons.concat(styles), {
+    styles = mergeTrees(addons.concat(styles), {
       overwrite: true,
       annotation: 'Styles',
     });
+    if (isExperimentEnabled('MODULE_UNIFICATION') && this.trees.src) {
+
+      // move only app/styles to /src/ui/styles
+      let styles1 = new Funnel(styles, {
+        allowEmpty: true,
+        srcDir: '/app/styles',
+        destDir: '/src/ui/styles',
+      });
+      let styles2 = new Funnel(styles, {
+        allowEmpty: true,
+        srcDir: '.',
+        exclude: ['app/styles/**'],
+        destDir: '.',
+      });
+      styles = mergeTrees([styles1, styles2], {
+        annotation: 'Styles MU',
+      });
+
+    }
+    return styles;
```

Comment:
Can you wrap the final return value in a `broccoli-debug` tree? It will make debugging this massively easier...

Reply:
https://github.com/ember-cli/ember-cli/pull/8289/commits/197aabe2dab76e2dbe9774692862a865118e3568